cornice_swagger/__init__.py

```diff
@@ -28,3 +29,42 @@ def includeme(config):
     config.add_view_predicate('tags', CorniceSwaggerPredicate)
     config.add_view_predicate('operation_id', CorniceSwaggerPredicate)
     config.add_view_predicate('api_security', CorniceSwaggerPredicate)
+    config.add_directive('register_swagger_ui', register_swagger_ui)
+
+
+def register_swagger_ui(
```

Comment:
maybe registering the`api_path` could be a separate directive, thus allowing one to register the api docs without using the api_explorer. What do you think?

Reply:
Sure, sounds like a sane change.

Reply:
@gabisurita any suggestions of what we want to name the other directive?